src/me/drton/flightplot/FlightPlot.java

```diff
@@ -729,6 +729,11 @@ public void showImportPresetDialog() {
                 Preset preset = Preset.unpackJSONObject(new JSONObject(new String(b, Charset.forName("utf8"))));
                 loadPreset(preset);
                 processFile();
+                if (preset.getTitle().equals("")) {
+                    // generate a namestring
+                    preset.setTitle(file.getName().substring(0, file.getName().indexOf(".fplot")));
+                }
+                presetComboBox.addItem(preset);
```

Comment:
If added to the list, one might make the assumption that this entry is somehow connected with the file you imported. I don't think it's good to mix these ATM. Maybe it needs a bit of rethinking in the direction of merging these 2 ways of handling presets.